package.json

```diff
@@ -46,6 +46,21 @@
 		"validate": "npm ls",
 		"version": "git add package.json && git commit -m \"Bump package version\""
 	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [
+			"npm run lint:fix",
+			"git add"
```

Comment:
Sorry for the delay.  Now that we're updated to Node 12, I had a chance to try this out, and I saw this error message: 
```
husky > pre-commit (node v12.18.3)
⚠ Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.
```
When I tried removing the `git add` line, it worked fine, so it's possible that it isn't needed (at least for me?).  Could you try it again without `git add` to see if it works for you?